newIDE/app/src/ObjectEditor/Editors/CustomObjectPropertiesEditor.js

```diff
@@ -150,13 +152,16 @@ const CustomObjectPropertiesEditor = (props: Props) => {
                               <EditorComponent
                                 objectConfiguration={childObjectConfiguration}
                                 project={project}
+                                layout={layout}
                                 resourceManagementProps={
                                   resourceManagementProps
                                 }
                                 onSizeUpdated={
                                   forceUpdate /*Force update to ensure dialog is properly positionned*/
                                 }
-                                objectName={childObject.getName()}
+                                objectName={
+                                  objectName + ' ' + childObject.getName()
```

Comment:
Is this a risky change?

Reply:
No, it's only used for default file names in Piskel (there is a comment about it somewhere).